client/src/views/pages/you-are-banned-what-now.js

```diff
@@ -0,0 +1,486 @@
+import StepWizard from 'react-step-wizard';
+import React, { useState } from 'react';
+
+import { useHistory } from 'react-router-dom';
+
+import styles from '../../assets/scss/custom/wizard.less';
```

Comment:
Is there a `sass` version of this? We should ideally convert it to sass and then import it in the main sass file so it's included in the main app and not in this specific component.